redash/plural/notes.tpl

```diff
@@ -0,0 +1 @@
+Use `plural watch redash` to track the status of your application
```

Comment:
we should add a line like:

`View your redash instance at {{ .Values.hostname }}`